app/views/admin/instances/show.html.haml

```diff
@@ -55,6 +55,24 @@
       = render partial: 'admin/action_logs/action_log', collection: @action_logs
     = link_to t('admin.instances.audit_log.view_all'), admin_action_logs_path(target_domain: @instance.domain), class: 'button'
 
+%hr.spacer/
+
+- if @instance.domain.present?
```

Comment:
@ClearlyClaire this is the work around I needed in the view due to the invalid instance domain coming through from the routing. I guess it's okay for now?